### PR TITLE
Add dad's feedback. Fixes #44.

### LINKED ISSUE #44: General fixes/feedback
My dad took a look and offered feedback. This is the easy/general stuff - there are a few other things that I'll put in other issues. I'm also going to use this issue to clean up a few parts of the text I found on a read-through.

### My dad's feedback

Introduction
- “straightforward” used twice.

Keep different parts of your life separate
- Formatting: hanging period after first sentence.
- sp. “acknowledgment” not “acknowledgement”

Pick services that are focused on one thing
- “…there’s less of a chance that the service will go away without warning.” Citation needed!!
  - Note from Eli: I don't know if there's a citation here that we can use, so maybe we change this so that it's clear that it's our opinion.)

Two-Factor Authentication
- “…we recommend using an authenticator app over text messages…” Not enough info here explaining what an authenticator app does as opposed to the text message approach – how does it look to the user? How does one set it up with a site that offers text TFA?, etc.

Internet of Things (IoT)
- Recommend adding home security systems and doorbell viewers to the list of examples.

Check the URL before Entering Sensitive Information
- Paranoia Alert!: I had to read the “Bonus Points…” sentence four times before I could decode it.  Suggest replacing “as” with “since” or “because”.

### PR DESCRIPTION
**Pull request by dispatchrabbi**

### CODE REVIEW

File: src/text.md [hunk 1]
```diff
@@ -109,11 +109,15 @@ Two-factor authentication (sometimes called Multi-Factor Authentication, or MFA)
...
 
-You'll also get a list of recovery codes when you set up 2FA. You won't be able to access them again later, so keep them safe and in a place where you can get to them without needing to log into an account that might require a recovery code. Storing printed copies in a safe is another good option.
+When setting up 2FA, if you are given the choice between using an authenticator app or receiving text messages, we recommend using the authenticator app.[^authapp] If the service can only send text messages, that's fine too. The authenticator app strategy is slightly more secure, but not enough that we're going to make a big deal out of it.
+
+  [^authapp]: Google also has a two-factor authentication system where, if you have an Android phone, it will pop up a screen on your phone asking if you're signing in. This only works for Google apps, but it's also a valid choice if you want to use it.
```
Comment (cajunluke) [inline src/text.md:118]: Apple has a similar service that pops up a code on iOS devices and Macs when you log into Apple services.

File: src/text.md [hunk 2]
```diff
@@ -279,22 +286,31 @@ We know that this is a hard line to take, and it pulls hard for the security sid
 ## Smart Speakers
 If you have a smart speaker, **throw it out**.
 
-Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we judge the potential tradeoffs to be too high. We judge it impossible to confidently and securely host a smart speaker in your house — so don't.
+Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we think the potential tradeoffs are too high. We possible to confidently and securely host a smart speaker in your house — so don't.
```
Comment (cajunluke) [inline src/text.md:289]:
```suggestion
Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we think the potential tradeoffs are too high. We are of the opinion that it is impossible to confidently and securely host a smart speaker in your house — so don't.
```

**Review by cajunluke — APPROVED** [review]

**Comment by cajunluke** [comment]
Don't forget to update the info.json edited date

File: src/text.md (hunk 2 above)
Comment (dispatchrabbi) [reply src/text.md:289]: I have no idea what the heck happened here.

**Pull request merged**